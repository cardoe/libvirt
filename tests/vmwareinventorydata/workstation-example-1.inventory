.encoding = "windows-1252"
vmlist6.config = "F:\Virtual Machines\NDCMIND2\NDCMIND2.vmx"
vmlist6.DisplayName = "NDCMIND2"
vmlist6.ParentID = "57"
vmlist6.ItemID = "6"
vmlist6.SeqID = "4"
vmlist6.IsFavorite = "FALSE"
vmlist6.IsClone = "FALSE"
vmlist6.CfgVersion = "8"
vmlist6.State = "normal"
vmlist7.config = "F:\Virtual Machines\NDCMIND3\NDCMIND3.vmx"
vmlist7.DisplayName = "NDCMIND3"
vmlist7.ParentID = "57"
vmlist7.ItemID = "7"
vmlist7.SeqID = "3"
vmlist7.IsFavorite = "FALSE"
vmlist7.IsClone = "FALSE"
vmlist7.CfgVersion = "8"
vmlist7.State = "normal"
vmlist8.config = "F:\Virtual Machines\NDCMIND3-OVM\NDCMIND3-OVM.vmx"
vmlist8.DisplayName = "NDCMIND3-OVM"
vmlist8.ParentID = "57"
vmlist8.ItemID = "8"
vmlist8.SeqID = "2"
vmlist8.IsFavorite = "FALSE"
vmlist8.IsClone = "FALSE"
vmlist8.CfgVersion = "8"
vmlist8.State = "normal"
vmlist9.config = "F:\Virtual Machines\AcronisESXAppliance\AcronisESXAppliance.vmx"
vmlist9.DisplayName = "AcronisESXAppliance"
vmlist9.ParentID = "58"
vmlist9.ItemID = "9"
vmlist9.SeqID = "0"
vmlist9.IsFavorite = "FALSE"
vmlist9.IsClone = "FALSE"
vmlist9.CfgVersion = "8"
vmlist9.State = "normal"
vmlist10.config = "H:\Virtual Machines\NDC2MIND\NDC2MIND-1.vmx"
vmlist10.DisplayName = "NDC2MIND-1"
vmlist10.ParentID = "57"
vmlist10.ItemID = "10"
vmlist10.SeqID = "1"
vmlist10.IsFavorite = "FALSE"
vmlist10.IsClone = "FALSE"
vmlist10.CfgVersion = "8"
vmlist10.State = "normal"
vmlist11.config = "J:\Virtual Machines\NDC2MIND-2\NDC2MIND-2.vmx"
vmlist11.DisplayName = "NDC2MIND-2"
vmlist11.ParentID = "57"
vmlist11.ItemID = "11"
vmlist11.SeqID = "0"
vmlist11.IsFavorite = "FALSE"
vmlist11.IsClone = "FALSE"
vmlist11.CfgVersion = "8"
vmlist11.State = "normal"
vmlist6.UUID = ""
vmlist6.IsCfgPathNormalized = "TRUE"
vmlist7.UUID = ""
vmlist7.IsCfgPathNormalized = "TRUE"
vmlist8.UUID = ""
vmlist8.IsCfgPathNormalized = "TRUE"
vmlist9.UUID = ""
vmlist9.IsCfgPathNormalized = "TRUE"
vmlist10.UUID = ""
vmlist10.IsCfgPathNormalized = "TRUE"
vmlist11.UUID = "56 4d 3d 15 4b e6 c5 01-f8 6f 58 da dd fc d8 51"
vmlist11.IsCfgPathNormalized = "TRUE"
vmlist48.config = "F:\Virtual Machines\maas-controller\maas-controller.vmx"
vmlist48.DisplayName = "maas-controller"
vmlist48.ParentID = "56"
vmlist48.ItemID = "48"
vmlist48.SeqID = "0"
vmlist48.IsFavorite = "FALSE"
vmlist48.IsClone = "FALSE"
vmlist48.CfgVersion = "8"
vmlist48.State = "normal"
vmlist48.UUID = "56 4d 65 d2 5f fd d4 94-ac 1c 7c e6 98 f7 40 47"
vmlist48.IsCfgPathNormalized = "TRUE"
vmlist49.config = "F:\Virtual Machines\maas-node0\maas-node0.vmx"
vmlist49.DisplayName = "maas-node0 (f8:12) tffyr"
vmlist49.ParentID = "56"
vmlist49.ItemID = "49"
vmlist49.SeqID = "1"
vmlist49.IsFavorite = "FALSE"
vmlist49.IsClone = "FALSE"
vmlist49.CfgVersion = "8"
vmlist49.State = "normal"
vmlist49.UUID = "56 4d 95 92 32 51 6e e3-8a 5f 53 bb ba f8 f8 f4"
vmlist49.IsCfgPathNormalized = "TRUE"
vmlist50.config = "F:\Virtual Machines\maas-node1\maas-node1.vmx"
vmlist50.DisplayName = "maas-node1 (b9:30) dkwem"
vmlist50.ParentID = "56"
vmlist50.ItemID = "50"
vmlist50.SeqID = "2"
vmlist50.IsFavorite = "FALSE"
vmlist50.IsClone = "FALSE"
vmlist50.CfgVersion = "8"
vmlist50.State = "normal"
vmlist50.UUID = "56 4d ac 9b eb ee 3c 60-0f ae 89 58 7d 28 b9 30"
vmlist50.IsCfgPathNormalized = "TRUE"
vmlist51.config = "F:\Virtual Machines\maas-node2\maas-node2.vmx"
vmlist51.DisplayName = "maas-node2 (b6:bc) amh7g"
vmlist51.ParentID = "56"
vmlist51.ItemID = "51"
vmlist51.SeqID = "3"
vmlist51.IsFavorite = "FALSE"
vmlist51.IsClone = "FALSE"
vmlist51.CfgVersion = "8"
vmlist51.State = "normal"
vmlist51.UUID = "56 4d ac 23 a6 5d 23 dd-a8 2d 69 29 d9 e1 b6 bc"
vmlist51.IsCfgPathNormalized = "TRUE"
vmlist52.config = "F:\Virtual Machines\maas-node3\maas-node3.vmx"
vmlist52.DisplayName = "maas-node3 (58:4d) x6f4p"
vmlist52.ParentID = "56"
vmlist52.ItemID = "52"
vmlist52.SeqID = "4"
vmlist52.IsFavorite = "FALSE"
vmlist52.IsClone = "FALSE"
vmlist52.CfgVersion = "8"
vmlist52.State = "normal"
vmlist52.UUID = "56 4d 5c 3c aa 3a bd da-0c 4e 9d 14 bc 23 58 4d"
vmlist52.IsCfgPathNormalized = "TRUE"
vmlist53.config = "F:\Virtual Machines\maas-node4\maas-node4.vmx"
vmlist53.DisplayName = "maas-node4 (a3:81) qj3wt"
vmlist53.ParentID = "56"
vmlist53.ItemID = "53"
vmlist53.SeqID = "5"
vmlist53.IsFavorite = "FALSE"
vmlist53.IsClone = "FALSE"
vmlist53.CfgVersion = "8"
vmlist53.State = "normal"
vmlist53.UUID = "56 4d 89 6a 67 69 69 1e-f2 c4 5e 2e 0e 49 a3 81"
vmlist53.IsCfgPathNormalized = "TRUE"
vmlist54.config = "F:\Virtual Machines\maas-node5\maas-node5.vmx"
vmlist54.DisplayName = "maas-node5 (36:e3) agmdh"
vmlist54.ParentID = "56"
vmlist54.ItemID = "54"
vmlist54.SeqID = "6"
vmlist54.IsFavorite = "FALSE"
vmlist54.IsClone = "FALSE"
vmlist54.CfgVersion = "8"
vmlist54.State = "normal"
vmlist54.UUID = "56 4d b5 a9 56 8c b4 6d-9a 3f a3 98 ee 32 36 e3"
vmlist54.IsCfgPathNormalized = "TRUE"
vmlist55.config = "F:\Virtual Machines\TEST-BUILD\TEST-BUILD.vmx"
vmlist55.DisplayName = "TEST-BUILD"
vmlist55.ParentID = "56"
vmlist55.ItemID = "55"
vmlist55.SeqID = "7"
vmlist55.IsFavorite = "FALSE"
vmlist55.IsClone = "FALSE"
vmlist55.CfgVersion = "8"
vmlist55.State = "normal"
vmlist55.UUID = "56 4d a7 96 8e 23 c0 aa-79 95 83 d9 66 83 7c e0"
vmlist55.IsCfgPathNormalized = "TRUE"
vmlist56.config = "folder56"
vmlist56.Type = "2"
vmlist56.DisplayName = "MAAS/OPENSTACK"
vmlist56.ParentID = "0"
vmlist56.ItemID = "56"
vmlist56.SeqID = "2"
vmlist56.IsFavorite = "FALSE"
vmlist56.UUID = "folder:52 61 29 e5 e1 45 61 11-ec b8 85 20 e8 69 5b 3f"
vmlist56.Expanded = "TRUE"
vmlist57.config = "folder57"
vmlist57.Type = "2"
vmlist57.DisplayName = "MIND"
vmlist57.ParentID = "0"
vmlist57.ItemID = "57"
vmlist57.SeqID = "1"
vmlist57.IsFavorite = "FALSE"
vmlist57.UUID = "folder:52 07 93 d1 c8 a0 94 4a-d0 a6 42 23 ec dc 8c 5c"
vmlist57.Expanded = "TRUE"
vmlist58.config = "folder58"
vmlist58.Type = "2"
vmlist58.DisplayName = "APPLIANCES"
vmlist58.ParentID = "0"
vmlist58.ItemID = "58"
vmlist58.SeqID = "0"
vmlist58.IsFavorite = "FALSE"
vmlist58.UUID = "folder:52 d6 21 02 23 40 55 ac-2f 18 b8 46 64 3d 40 b1"
vmlist58.Expanded = "TRUE"
vmlist1.config = ""
vmlist2.config = ""
vmlist3.config = ""
vmlist4.config = ""
vmlist5.config = ""
vmlist12.config = ""
vmlist13.config = ""
vmlist14.config = ""
vmlist15.config = ""
vmlist16.config = ""
vmlist17.config = ""
vmlist18.config = ""
vmlist19.config = ""
vmlist20.config = ""
vmlist21.config = ""
vmlist22.config = ""
vmlist23.config = ""
vmlist24.config = ""
vmlist25.config = ""
vmlist26.config = ""
vmlist27.config = ""
vmlist28.config = ""
vmlist29.config = ""
vmlist30.config = ""
vmlist31.config = ""
vmlist32.config = ""
vmlist33.config = ""
vmlist34.config = ""
vmlist35.config = ""
vmlist36.config = ""
vmlist37.config = ""
vmlist38.config = ""
vmlist39.config = ""
vmlist40.config = ""
vmlist41.config = ""
vmlist42.config = ""
vmlist43.config = ""
vmlist44.config = ""
vmlist45.config = ""
vmlist46.config = ""
vmlist47.config = ""
vmlist59.config = "F:\Virtual Machines\maas-test-node\maas-test-node.vmx"
vmlist59.DisplayName = "maas-test-node"
vmlist59.ParentID = "0"
vmlist59.ItemID = "59"
vmlist59.SeqID = "3"
vmlist59.IsFavorite = "FALSE"
vmlist59.IsClone = "FALSE"
vmlist59.CfgVersion = "8"
vmlist59.State = "normal"
vmlist59.UUID = "56 4d b6 99 58 9d 91 e7-5b 40 15 33 f8 b3 e9 03"
vmlist59.IsCfgPathNormalized = "TRUE"
index0.field0.name = "guest"
index0.field0.value = "other"
index0.field0.default = "TRUE"
index0.field1.name = "notes"
index0.field1.value = "{ab53a0f1-ad54-480f-80bb-fc72dc41df53}"
index0.field1.default = "TRUE"
index0.hostID = "localhost"
index0.id = "F:\Virtual Machines\AcronisESXAppliance\AcronisESXAppliance.vmx"
index0.field.count = "2"
index1.field0.name = "guest"
index1.field0.value = "centos 64-bit"
index1.field0.default = "TRUE"
index1.hostID = "localhost"
index1.id = "F:\Virtual Machines\NDCMIND2\NDCMIND2.vmx"
index1.field.count = "1"
index2.field0.name = "guest"
index2.field0.value = "centos 64-bit"
index2.field0.default = "TRUE"
index2.hostID = "localhost"
index2.id = "F:\Virtual Machines\NDCMIND3-OVM\NDCMIND3-OVM.vmx"
index2.field.count = "1"
index3.field0.name = "guest"
index3.field0.value = "centos 64-bit"
index3.field0.default = "TRUE"
index3.hostID = "localhost"
index3.id = "F:\Virtual Machines\NDCMIND3\NDCMIND3.vmx"
index3.field.count = "1"
index4.field0.name = "guest"
index4.field0.value = "ubuntu 64-bit"
index4.field0.default = "TRUE"
index4.hostID = "localhost"
index4.id = "F:\Virtual Machines\TEST-BUILD\TEST-BUILD.vmx"
index4.field.count = "1"
index5.field0.name = "guest"
index5.field0.value = "ubuntu 64-bit"
index5.field0.default = "TRUE"
index5.hostID = "localhost"
index5.id = "F:\Virtual Machines\maas-controller\maas-controller.vmx"
index5.field.count = "1"
index6.field0.name = "guest"
index6.field0.value = "ubuntu 64-bit"
index6.field0.default = "TRUE"
index6.hostID = "localhost"
index6.id = "F:\Virtual Machines\maas-node0\maas-node0.vmx"
index6.field.count = "1"
index7.field0.name = "guest"
index7.field0.value = "ubuntu 64-bit"
index7.field0.default = "TRUE"
index7.hostID = "localhost"
index7.id = "F:\Virtual Machines\maas-node1\maas-node1.vmx"
index7.field.count = "1"
index8.field0.name = "guest"
index8.field0.value = "ubuntu 64-bit"
index8.field0.default = "TRUE"
index8.hostID = "localhost"
index8.id = "F:\Virtual Machines\maas-node2\maas-node2.vmx"
index8.field.count = "1"
index9.field0.name = "guest"
index9.field0.value = "ubuntu 64-bit"
index9.field0.default = "TRUE"
index9.hostID = "localhost"
index9.id = "F:\Virtual Machines\maas-node3\maas-node3.vmx"
index9.field.count = "1"
index10.field0.name = "guest"
index10.field0.value = "ubuntu 64-bit"
index10.field0.default = "TRUE"
index10.hostID = "localhost"
index10.id = "F:\Virtual Machines\maas-node4\maas-node4.vmx"
index10.field.count = "1"
index11.field0.name = "guest"
index11.field0.value = "ubuntu 64-bit"
index11.field0.default = "TRUE"
index11.hostID = "localhost"
index11.id = "F:\Virtual Machines\maas-node5\maas-node5.vmx"
index11.field.count = "1"
index12.field0.name = "guest"
index12.field0.value = "ubuntu 64-bit"
index12.field0.default = "TRUE"
index12.hostID = "localhost"
index12.id = "F:\Virtual Machines\maas-test-node\maas-test-node.vmx"
index12.field.count = "1"
index13.field0.name = "guest"
index13.field0.value = "centos 64-bit"
index13.field0.default = "TRUE"
index13.hostID = "localhost"
index13.id = "H:\Virtual Machines\NDC2MIND\NDC2MIND-1.vmx"
index13.field.count = "1"
index14.field0.name = "guest"
index14.field0.value = "centos 64-bit"
index14.field0.default = "TRUE"
index14.hostID = "localhost"
index14.id = "J:\Virtual Machines\NDC2MIND-2\NDC2MIND-2.vmx"
index14.field.count = "1"
index.count = "15"
