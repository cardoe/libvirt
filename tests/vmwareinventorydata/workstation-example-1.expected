F:\Virtual Machines\NDCMIND2\NDCMIND2.vmx
F:\Virtual Machines\NDCMIND3\NDCMIND3.vmx
F:\Virtual Machines\NDCMIND3-OVM\NDCMIND3-OVM.vmx
F:\Virtual Machines\AcronisESXAppliance\AcronisESXAppliance.vmx
H:\Virtual Machines\NDC2MIND\NDC2MIND-1.vmx
J:\Virtual Machines\NDC2MIND-2\NDC2MIND-2.vmx
F:\Virtual Machines\maas-controller\maas-controller.vmx
F:\Virtual Machines\maas-node0\maas-node0.vmx
F:\Virtual Machines\maas-node1\maas-node1.vmx
F:\Virtual Machines\maas-node2\maas-node2.vmx
F:\Virtual Machines\maas-node3\maas-node3.vmx
F:\Virtual Machines\maas-node4\maas-node4.vmx
F:\Virtual Machines\maas-node5\maas-node5.vmx
F:\Virtual Machines\TEST-BUILD\TEST-BUILD.vmx
F:\Virtual Machines\maas-test-node\maas-test-node.vmx
